Homework4/Alex_Zubok_home_work4_1_sorts.cpp

```diff
@@ -0,0 +1,52 @@
+#include <cstring>
+#include <iostream>
+
+using Comp = bool (*)(int, int);
+bool More(int a, int b) { return a > b; }
+bool Less(int a, int b) { return a < b; }
```

Comment:
```suggestion
bool More(int a, int b) { return a > b; }
bool Less(int a, int b) { return a < b; }
using Comp = decltype(&More);
```
 А зачем вам Less если она не используется?

Reply:
убрал лесс, она была, для проверки вместо лямбды